Cargo.toml

```diff
@@ -87,6 +87,7 @@ codegen-units = 1
 
 [patch.crates-io]
 hdwallet = { git = "https://github.com/nuttycom/hdwallet", rev = "576683b9f2865f1118c309017ff36e01f84420c9" }
+incrementalmerkletree = { git = "https://github.com/nuttycom/incrementalmerkletree", rev = "fc1d1ab9fc988c8c16a6713f80c1809dfbb697d0" }
```

Comment:
This doesn't match the current tip of https://github.com/zcash/incrementalmerkletree/pull/31.